sympy/stats/stochastic_process_types.py

```diff
@@ -326,6 +324,104 @@ def _preprocess(self, given_condition, evaluate):
 
         return is_insufficient, trans_probs, state_space, given_condition
 
+    def _transient2transient(self):
+        """
+        Computes the one step probabilities of transient
+        states to transient states. Used in finding
+        fundamental matrix, absorbing probabilties.
+        """
+        trans_probs = self.transition_probabilities
+        if not isinstance(trans_probs, ImmutableMatrix):
+            return None
+
+        m, trans_states, t2t = trans_probs.shape[0], [], []
+        for i in range(m):
+            if trans_probs[i, i] != 1:
+                trans_states.append(i)
+
+        for si in trans_states:
+            tij = []
+            for sj in trans_states:
+                tij.append(trans_probs[si, sj])
+            t2t.append(tij)
+
+        return ImmutableMatrix(t2t)
+
+    def _transient2absorbing(self):
+        """
+        Computes the one step probabilities of transient
+        states to absorbing states. Used in finding
+        fundamental matrix, absorbing probabilties.
+        """
+        trans_probs = self.transition_probabilities
+        if not isinstance(trans_probs, ImmutableMatrix):
+            return None
+
+        m, trans_states, absorb_states, t2a = \
+            trans_probs.shape[0], [], [], []
+        for i in range(m):
+            if trans_probs[i, i] == 1:
+                absorb_states.append(i)
+            else:
+                trans_states.append(i)
+
+        if not absorb_states or not trans_states:
+            return None
+
+        for si in trans_states:
+            tij = []
+            for sj in absorb_states:
+                tij.append(trans_probs[si, sj])
+            t2a.append(tij)
+
+        return ImmutableMatrix(t2a)
+
+    @property
+    def fundamental_matrix(self):
+        Q = self._transient2transient()
+        if Q == None:
+            return None
+        I = eye(Q.shape[0])
+        if (I - Q).det() == 0:
+            raise ValueError("Fundamental matrix doesn't exists.")
+        return (I - Q).inv()
+
+    def is_absorbing_state(self, state):
+        trans_probs = self.transition_probabilities
+        if isinstance(trans_probs, ImmutableMatrix) and \
+            state < trans_probs.shape[0]:
+            return S(trans_probs[state, state]) == S.One
+
+    @property
+    def is_absorbing_chain(self):
+        trans_probs = self.transition_probabilities
+        return any(self.is_absorbing_state(state) == True
+                    for state in range(trans_probs.shape[0]))
+
+    @property
```

Comment:
if it's something you need to compute, maybe it's better to avoid setting it as a property.